src/components/Configs/ConfigList/ConfigListColumn.tsx

```diff
@@ -107,20 +118,28 @@ export const configListColumns: ColumnDef<ConfigItem, any>[] = [
   //     return <span>{agent?.name}</span>;
   //   }
   // },
+  // {
+  //   header: "Tags",
+  //   accessorKey: "tags",
+  //   cell: React.memo(ConfigListTagsCell),
+  //   aggregatedCell: "",
+  //   size: 240
+  // },
+  // {
+  //   header: "Tags",
+  //   accessorKey: "tags",
+  //   cell: React.memo((props) => (
+  //     <ConfigListTagsCell {...props} hideGroupByView />
+  //   )),
+  //   aggregatedCell: "",
+  //   size: 240
+  // },
   {
-    header: "Tags",
-    accessorKey: "tags",
-    cell: React.memo(ConfigListTagsCell),
-    aggregatedCell: "",
-    size: 240
-  },
-  {
-    header: "All Tags",
-    accessorKey: "allTags",
+    header: "Labels",
+    accessorKey: "labels",
```

Comment:
We should be returning or showing labels, only tags , and they should be unlabelled, if the tag is used for grouping, it shouldn't be shown either